monitoring/osd-configuring-the-monitoring-stack.adoc

```diff
@@ -26,6 +26,8 @@ include::modules/osd-monitoring-moving-monitoring-components-to-different-nodes.
 * link:https://docs.openshift.com/container-platform/4.7/nodes/scheduling/nodes-scheduler-node-selectors.html[Placing pods on specific nodes using node selectors]
 * See the link:https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector[Kubernetes documentation] for details on the `nodeSelector` constraint
 
+// TODO: Same question about whether links out to OCP docs should explicitly mention that? (I won't point out any more, but could easily search to find other instances in the repo).
```

Comment:
The end goal is to pull in the related OCP content and we're actively working with PM to figure out what pieces get pulled in. JQ wrote this before we had that capability, hence the linking out to OCP.